source/MaterialXCore/Node.cpp

```diff
@@ -714,6 +722,18 @@ NodeDefPtr NodeGraph::getNodeDef() const
             }
         }
     }
+    // Check datalibrary if available
```

Comment:
Again should this not come first for consistency ?

Reply:
When do an import aren't the library nodes added before the current document? In that case we are keeping the same logic checking the library nodes and then local. Let me know if this should be still reversed.